### DISCUSSION TITLE
Have a NAME.DATE ??

### DISCUSSION THREAD

**Discussion by mother10** [FamilySearch/GEDCOM#471] [Ideas]
Would it be possible to add a DATE to a NAME tag? Lots of names in older records change during lifetime of a person. It could be becaue of misspelling, but also other reasons.
To me it would look handy to have a DATE with a NAME tag. That way you could enter the NAME and add the date of the event you found that NAME spelling. That might help to sort Name changes. Or find misspellings that turned into a name change and such.
It could also help maybe in figuring out of maybe 2 persons are in fact 1, or the other way around.

I know you could see the date of a name by going to the events where a NAME is added, but if you find a different spelling in a source, you would have to input tyhe new name and write a NOTE to that NAME and put in there what source and when you found this NAME spelling. But you would still not be able to sort the names then.

As it is now, there is no connection between a source and a certain name spelling in the gedcom.

**Comment by dthaler** [comment]
Discussion during GEDCOM Steering Committee meeting:
* In general we agree that something like this is needed
* We think this has been discussed in the past and even put into a draft proposal somewhere that never made it in yet.  This is often asked for but we haven't gotten reports about any apps currently doing this.
* More input and implementation experience was desired from the community, but this is a good candidate for inclusion in 7.1

**Reply by mother10** [reply]
@dthaler  I would suggest you look here: [https://github.com/FamilySearch/GEDCOM/issues/473](url)
Because in that discussion it was proposed to maybe have NAME on onother place in the Gedcom.
I now think NAME should be part of an event, not part of an INDI. The fact that names change BECAUSE of en event, proves to me it belongs there logically.
And the fact people start asking for more nametypes, all related to an event, proves that too.

So we should not have:
0 INDI
1 NAME ThisIsMyName WithUnknownType

But instead:
0 INDI
1 BIRT
2 NAME ThisIsMyBirthName
2 DATE BirthDate
2 PLAC x,y,z

0 INDI
1 ADOP
2 NAME ThisIsMyAdoptiveName
2 PLAC x,y,z
2 DATE AdoptionDate

Etcetera. (look in that discussion for the whole reasoning) Nametype would then no longer be necessary!!
The fact people start asking about more and more Nametypes, to me, now, after that other discussion, is a prove that we might be on the wrong track.

So even when it was me, who started this subject about DATE for a NAME, I would now say: Dont just do that, but to get in touch there. because if indeed NAME would be shifted in version 8, this DATE I proposed here, would no longer be necesary.
So maybe better to discuss that there first, than now do work that might be changed back again in version 8.
We dont want to do unnecessary work do we?

**Comment by mother10** [comment]
As this is added to the proposed NAME structure, shouldn't this be closed then?
